restli-tools/src/test/java/com/linkedin/restli/tools/data/TestExtensionSchemaValidationCmdLineApp.java

```diff
@@ -89,12 +89,17 @@ public void testExtensionSchemaValidation(String inputDir, boolean isValid, Stri
       try
       {
         ExtensionSchemaValidationCmdLineApp.parseAndValidateExtensionSchemas(resolverPath, new File(inputPath));
-        Assert.assertTrue(isValid);
+        SoftAssert softAssert = new SoftAssert();
+        softAssert.assertTrue(isValid);
+        softAssert.assertEquals(null, errorMessage);
+        softAssert.assertAll();
       }
       catch (Exception e)
       {
-        Assert.assertTrue(!isValid);
-        Assert.assertEquals(e.getMessage(), errorMessage);
+        SoftAssert softAssert = new SoftAssert();
+        softAssert.assertTrue(!isValid);
+        softAssert.assertEquals(e.getMessage(), errorMessage);
+        softAssert.assertAll();
       }
```

Comment:
It would make more sense to me to create a single `SoftAssert` and then `assertAll()` at the end:

```
SoftAssert softAssert = new SoftAssert();

try
{
  ExtensionSchemaValidationCmdLineApp.parseAndValidateExtensionSchemas(resolverPath, new File(inputPath));
  softAssert.assertTrue(isValid);
  softAssert.assertEquals(null, errorMessage);
}
catch (Exception e)
{
  softAssert.assertTrue(!isValid);
  softAssert.assertEquals(e.getMessage(), errorMessage);
}

softAssert.assertAll();
```

Otherwise, looks good to me.

Reply:
Ah yes, I can make that change